Dockerfile.external

```diff
@@ -14,27 +13,48 @@ FROM base as dependencies
 WORKDIR /home/worker/app
 COPY pyproject.toml poetry.lock ./
 
-RUN poetry install --extras "ui vector-stores-qdrant llms-ollama embeddings-ollama"
+ARG POETRY_EXTRAS="ui vector-stores-qdrant llms-ollama embeddings-ollama"
+RUN poetry install --no-root --extras "${POETRY_EXTRAS}"
 
 FROM base as app
-
-ENV PYTHONUNBUFFERED=1
-ENV PORT=8080
+ENV \
+  PYTHONUNBUFFERED=1 \
+  PORT=8080 \
+  PYTHONPATH="$PYTHONPATH:/home/worker/app/private_gpt/" \
+  APP_ENV=prod \
+  PGPT_PROFILES=docker \
+  PGPT_MODE=mock \
+  PGPT_EMBEDDING_MODE=sagemaker \
+  PGPT_HF_REPO_ID=TheBloke/Mistral-7B-Instruct-v0.1-GGUF \
+  PGPT_HF_MODEL_FILE=mistral-7b-instruct-v0.1.Q4_K_M.gguf \
+  PGPT_EMBEDDING_HF_MODEL_NAME=BAAI/bge-small-en-v1.5 \
+  PGPT_SAGEMAKER_LLM_ENDPOINT_NAME= \
+  PGPT_SAGEMAKER_EMBEDDING_ENDPOINT_NAME= \
+  PGPT_OLLAMA_LLM_MODEL=mistral \
+  PGPT_OLLAMA_EMBEDDING_MODEL=nomic-embed-text \
+  PGPT_OLLAMA_API_BASE=http://ollama:11434 \
+  PGPT_OLLAMA_TFS_Z=1.0 \
+  PGPT_OLLAMA_TOP_K=40 \
+  PGPT_OLLAMA_TOP_P=0.9 \
+  PGPT_OLLAMA_REPEAT_LAST_N=64 \
+  PGPT_OLLAMA_REPEAT_PENALTY=1.2 \
+  PGPT_OLLAMA_REQUEST_TIMEOUT=600.0
```

Comment:
I would avoid defining all the variables that might exist in the future. This will grow exponentially with the number of implementations we make. Additionally, duplicating the default values can lead to problems when someone runs Docker versus locally.